GWD

```diff
@@ -0,0 +1,42 @@
+Glenn Website Design
+http://glennwebsitedesign.com
+Company 
+Glenn creates custom websites. Glenn is American managing a remote company internet web design company from Indonesia.
+
+Company size - 0 - 20
+
+
+Remote status
+WE hire mostly remote based freelancers from http://freelancer.com but have open door employment.
```

Comment:
We only hire remote employee as we do not operate out of our office. I manage the business but I am on the road most of the time myself for various business purposes so I also manage the business remotely most of the year.

Reply:
Thanks for confirming.

Reply:
I created a new file in the company-profiles folder glennwebsitedesign.md
Then I add a new line to the main README.md file in alphabetical order.

I did not know how to delete or move the GWD file I created ???